### ISSUE TITLE
Allow changing of "launched version"

### ISSUE THREAD

**Issue by Jefaxe** [MultiMC/MultiMC5#3700]
### Role

Its nice to have custom version names.

### Suggestion

Supply an option in instance.cfg called "Launched Version". 

### Benefit

Its nice to have custom names, and its a bit weird to have every version called "MultiMC5".

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

In the Default Launcher, you can renamed the instance in launcher_profiles.json, then renamed the folder, jar, json, and json contents to change the version (shows up in F3).

**Comment by phit** [comment]
could you elaborate a bit what the actual benefit actually is? is that text used for anything?

**Comment by Jefaxe** [comment]
Yes, when you open F3 you can see it near the top. In vanilla its something like 1.16.5/vanilla, (1.16.5 is the version, vanilla the client brand). Also in the log it says "Launched Version: MultiMC5" for every version.

**Comment by Janrupf** [comment]
I would argue that this causes more problems than it solves - including but not limited to possible data inconsistency.

**Comment by Jefaxe** [comment]
yeah its just cosmetic, and fair enough if it just causes more issues. It'd be nice, but shall i just close this then?

**Comment by peterix** [comment]
Revisiting this is probably a good idea.

I've seen this misused by server operators to ban people (forcing screen capture to play and similar).

**Comment by AverageComet250** [comment]
Some anticheats use it to make sure that the user isn't running a hacked client. Normally, they make sure it's set to optifine or vanilla, and if it isn't, they just don't let the user connect.

It may be that it's picking up multimc and declaring it a hack client cause it's not the official launcher

Thanks,
Shravan Mandava

________________________________

**Comment by kb-1000** [comment]
Then those are buggy and should not be used.

**Comment by Jefaxe** [comment]
No, that's the Client Brand as far as I know, and optifine doesn't change the client brand, it just keeps it Vanilla. Also literally every single hacked client ever uses Vanilla Brand, so they aren't caught easily. "Launched version" data doesn't even get sent to the server, as far as I know.
